CMSgov/mint-app
# Easi 2573/possible operational solution treat as custom

# [EASI-2573](https://jiraent.cms.gov/browse/EASI-2573), [EASI-2502](https://jiraent.cms.gov/browse/EASI-2502), [EASI-2752](https://jiraent.cms.gov/browse/EASI-2752)

## Changes and Description

- 2573
    - Update Operational Solutions to have the ability to treat certain defined options as Other (eg Contractor)
    - Updated Possible Operational Solutions to set the correct ones as treat as Other
    - Added isOther and otherHeader to OperationalSolution
        - Custom solutions and defined treat as other solutions return isOther true
    - Removed addOrUpdate solution
       - Discrete mutation to insert
       - Discrete mutation to update
       - Remove legacy  update / insert mutation
  
- 2502
   - Update Operational Solutions to match the options in FIGMA and google sheets
   -  Updated Operational Solution links to match what is provided in the spreadsheet.
   
   
  Front End
   - Changes made in https://github.com/CMSgov/mint-app/pull/532
   



## How to test this change
Backend
1. Use the updated mutations in the postman collection to verify the new resolvers
2. Verify that the need solution links match google sheets / figma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
